### PR TITLE
UI for surface view

### PR DESCRIPTION
**Pull request by H3llAbyss**
Made by mr. Cherepovskyi and @H3llAbyss

### CODE REVIEW

File: Ui/CustomWidgets/CustomWidgets.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+class CreateCylindricalWidget(QtWidgets.QWidget):
+    def __init__(self):
+        super(CreateCylindricalWidget, self).__init__()
+        uic.loadUi("CustomWidgets\CreateCylindricalWidget.ui", self)
+
+
+class CreateLineWidget(QtWidgets.QWidget):
+    def __init__(self):
+        super(CreateLineWidget, self).__init__()
+        uic.loadUi("CustomWidgets\CreateLineWidget.ui", self)
```
Comment (DenDen039) [inline Ui/CustomWidgets/CustomWidgets.py:35]: Add support for linux path

**Comment by DenDen039** [comment]
Add git attributes

**Comment by H3llAbyss** [comment]
image
image
image

File: Ui/CustomWidgets/CustomWidgets.py (hunk 1 above)
Comment (DenDen039) [inline Ui/CustomWidgets/CustomWidgets.py:35]: Write makefile to generate python code. Instead of doing it dynamicly here

**Review by DenDen039 — CHANGES_REQUESTED** [review]

**Comment by DenDen039** [comment]
Why toolbox is separate window?

File: Ui/CustomWidgets/CustomWidgets.py (hunk 1 above)
Comment (H3llAbyss) [reply Ui/CustomWidgets/CustomWidgets.py:35]: Why

Comment (DenDen039) [reply Ui/CustomWidgets/CustomWidgets.py:35]: 1. This pathes doesn't work for linux
2. We can only get buttons by string name
3. It is not possible modify any code without qteditor

**Comment by H3llAbyss** [comment]
image

**Pull request merged**